content/modules/redisgears/release-notes/redisgraph-1.0-release-notes.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: RedisGears 1.0 Release Notes
+description:
+weight: 100
+alwaysopen: false
+categories: ["Modules"]
+---
+## RedisGears 1.0.0
+
+### Highlights
+
+* Thread pool support for running several executions in parallel.
+* Enhanced dependency installation and module configuration options.
+* Revision of all Readers with added arguments which results in more granular reading.
+
+See the [complete RedisGears 1.0.0 release notes on GitHub](https://github.com/edx/edx-platform/pull/13903).
```

Comment:
why would we do this @bmansheim the entire idea is to have them here

Reply:
Agreed. I forgot to tell @banker that. I'll fix it.